docs(release): v0.8.0 release notes + removal of stable tags (#551)
flybywiresim/docs: docs/release-notes/v080.md

# Stable Release v0.8.0

<link rel="stylesheet" href="../../stylesheets/toc-tables.css">

> It's been a long road to our latest stable release, and we are happy to introduce some long awaited features, stunning visual elements, and even better realistic simulation of 

Reviewer: ```suggestion
It has been a long road to our latest stable release, and we are happy to introduce some long awaited features, stunning visual elements, and even better realistic simulation of 
```
Eliminate contraction

A320neo systems. This rather large feature release even contains our first implementation of using onboard systems externally on your mobile devices. FBW pilots, "Cleared for 
Takeoff"!

This release now includes these major features:

- [x] Custom Flight Management System with huge improvements to LNAV capability
- [x] Hydraulics Overhaul
    - [x] **NEW** Separated Tiller 
- [x] CPDLC Hoppie ACARS
- [x] TCAS 7.1 and AP/FD TCAS
- [x] Cockpit Textures
- [x] MDCU Web Interface
- [x] Display Typeface Improvements
- [x] AP, FBW, and A/THR Improvements
- [x] Various system improvements and optimizations

For a full release changelog - [see here](#changelog)

!!! warning "Important User Experience Changes"
    Please note the following changes:

    - Custom Flight Management System
        - See the [Special Notes Section](../fbw-a32nx/feature-guides/cFMS.md#special-notes) on our custom FMS page for details on the topics below:
            - Weather and Terrain are now inoperable as we wait for Asobo implementations.
            - MSFS Built-in ATC and VFR maps are on limited support.

>     - Discontinuities are a feature and not a bug!

Reviewer: ```suggestion
    - Discontinuities may now appear in your flight plan—they are a feature and not a bug.
```
Fix logical flow and tone

|                       [Separated Tiller](#separated-tiller)                       |
|                     [CPDLC Hoppie ACARS](#cpdlc-hoppie-acars)                     |
|                                   [TCAS](#tcas)                                   |
|                       [Cockpit Textures](#cockpit-textures)                       |
|                     [MCDU Web Interface](#mcdu-web-interface)                     |
| [Various Improvements and Optimizations](#various-improvements-and-optimizations) |

---

## Custom Flight Management System

A crucial part of every IFR flight is planning your route, departures, and arrivals. Our new custom flight management focuses on bringing the system architecture even closer to 
how the real A320 handles FMS data and the interactions.

We have completed major improvements to the LNAV portion of the custom FMS. This in turn has allows the FMS to have full authority on drawing the flight plan. With a wide 
support for various procedure leg types you should experience flight paths that are more accurate to real life lateral SID/STAR constraints encoded in the navdata, better 
flight path prediction, and *finally* programmable holds!

> Additionally, we've also included the following features:

Reviewer: ```suggestion
Additionally, we have also included the following features:
```
Eliminate contraction

- [Fix Info](../pilots-corner/advanced-guides/flight-planning/fixinfo.md) to create waypoint intersections in your flight plan.
- [Stored Waypoints](../pilots-corner/advanced-guides/data-management.md) to enter and store up to 99 custom waypoints.
- [Discontinuities](../pilots-corner/advanced-guides/flight-planning/disco.md) are inserted into the F-PLN accordingly.

[Custom FMS Documentation](../fbw-a32nx/feature-guides/cFMS.md){.md-button}

![FMS Comparison](assets/comparisonfms1.png){loading=lazy}

![FMS Comparison 2](assets/comparsionfms2.png){loading=lazy}

## Hydraulics

A large amount of work has been done to create and implement a realistic simulation of various hydraulic systems in the A32NX. This update features all control surfaces hydraulic control loop (visual and hydraulic only), leak measurement valves, a physics based PTU model, emergency hydraulic motor/generator simulation supplied by RAT (no 3D model yet), and even our first iteration of the custom bleed system, impacting hydraulic reservoir pressurisation. 

> You may even notice the aileron droop when the aircraft is unpowered at the gate commonly known as "aileron droop". This even means that heavy winds on the ground can displace the rudder visually when unpowered, while the aerodynamic model can affect moving parts.

Reviewer: ```suggestion
You may even notice the aileron droop when the aircraft is unpowered at the gate. In addition, heavy winds on the ground can now displace the rudder visually when the aircraft is unpowered. This is because the aerodynamic model can now affect moving parts.
```
Fix logical flow

> As a part of the hydraulic update we have also made the ground handling much more realistic and have finally implemented steering via the [tiller](#separated-tiller).

Reviewer: ```suggestion
As part of the hydraulic update, we have also made ground handling much more realistic and have finally implemented steering via the [tiller](#separated-tiller).
```
Remove unnecessary articles and add comma

!!! tip "Deep Dive"
    Check out our dedicated [Hydraulics Development Update NOTAM](https://flybywiresim.com/notams/hydraulics-update-2022/){target=new} that explains a lot of these changes in detail.

### Separated Tiller

> Previously most users opt to use the rudder axis to steer the aircraft on the ground. Now nose wheel steering can now be accomplished via the tiller! This handwheel is utilized to 

Reviewer: ```suggestion
Previously, most users opted to use the rudder axis to steer the aircraft on the ground. Now, nose wheel steering can be accomplished via the tiller! It is used to
```
Add commas, fix tense, remove redundant wording and improve word choice

> turn the nose wheel during taxi up to 75° to the left or right creating a more realistic taxi experience in the simulator.

Reviewer: ```suggestion
turn the nose wheel during taxi up to 75° to the left or right, giving users a more realistic taxi experience in the simulator.
```
Add comma, fix logical flow

![NW Tiller](../fbw-a32nx/assets/nw-tiller/side-console-handwheel.png){loading=lazy}

[Nose Wheel Tiller Operation Guide](../fbw-a32nx/feature-guides/nw-tiller.md){.md-button}

## CPDLC Hoppie ACARS

Trying to get a non-urgent message on a VATSIM/IVAO but there are too many critical instructions on frequency? Use our CPDLC implementation! While this is a constant "in 

> development" feature it currently supports both networks for relaying two-way data-linked messages as an alternative to voice communications. 

Reviewer: ```suggestion
development" feature, it currently supports both networks for relaying two-way data-linked messages as an alternative to voice communications. 
```
Add comma

This entire implementation is a realistic realization of the ATS 623 standard.

![dcdu after send](../fbw-a32nx/assets/feature-guides/hoppie/dcdu-dcl-after-send.png){loading=lazy}

[Hoppie ACARS Operation Guide](../fbw-a32nx/feature-guides/hoppie.md){.md-button}

## TCAS

Our implementation of the Traffic Collision Avoidance System (TCAS) is based on TCAS 7.1 a first among all Airbus simulation addons. This includes the AP/FD TCAS feature that 
provides vertical guidance as part of the Auto Flight System (AFS).

> The A32NX can detect TCAS TAs and provide autopilot guided TCAS RA maneuvers on both VATSIM/IVAO networks and the live traffic feature in MSFS. *It does not support MSFS 

Reviewer: ```suggestion
The A32NX can detect TCAS TAs and provide autopilot guided TCAS RA manoeuvres on both VATSIM/IVAO networks and the live traffic feature in MSFS. *It does not support MSFS 
```
Use British spelling

- Improved engagement of OP CLB/DES modes (in coordination with ATHR)
- Improved SRS engagement conditions
- Removed V/S sub mode for CLB, OP CLB and DES when change is less than 1200 ft
- Improved SPD/MACH law

### ATHR

#### New
- Thrust limits redone
- Added distinct retard mode for Autoland (that uses a different N1 rate than THR IDLE)
- FLX thrust is now limiting CLB on entry, realistic transition to default CLB after take-off

#### Improved / Fixed
- Improved SPEED law that uses load-factor and improves scenarios with low N1 when flying transition from descend to level off
- Improved THR CLB and THR IDLE modes
- Improved SOFT ALT mode
- Improved accuracy to achieve N1 target (N1 control base loop)

### FBW

#### New
- Reworked roll law
- Reworked yaw law for side slip control and turn coordination
- Pitch attitude and load factor limits are now also applied in High-AoA protection

#### Improved / Fixed
- Improved pitch law
  - Increased pitch rate of C* law
  - Improved normal law flight path stability when speed is increased or decreased
- Improved stability in turbulent weather
- Improved FLARE mode (manual flight)

## Various Improvements and Optimizations

> We improved almost every aspect of the A32NX from our previous release v0.7.0. Features you already enjoy such as the flyPadOS 2 has seen changes and improvements to UI 

Reviewer: ```suggestion
We have improved almost every aspect of the A32NX from our previous release (v0.7.0). Features you already enjoy such as the flyPadOS 2 have seen changes and improvements to UI 
```
Fix tense, chenge singular > plural

> elements and how the data is handled. We've modeled cabin temperature control and simulation while also improving the pack flow model. Our external lighting systems have been 

Reviewer: ```suggestion
elements and how the data is handled. We have modeled cabin temperature control and simulation while also improving the pack flow model. Our external lighting systems have been 
```
Eliminate contraction

improved to have better logic, ambience, and are synced in their behaviors.

> We've been able to edit aspects of the A320neo model. While not a complete rehaul we've been able to add new texturing, decal and livery improvements for livery creators, heat 

Reviewer: ```suggestion
We have been able to edit aspects of the A320neo model. While not a complete overhaul, we've been able to add new texturing, decal and livery improvements for livery creators, heat 
```
Eliminate contraction, improve word choice, add comma

> effect changes on the engines, and even landing lights that retract fully and independently. We've also finally allowed the first officer to use the flyPad.

Reviewer: ```suggestion
effect changes on the engines, and even landing lights that retract fully and independently. We have also finally allowed the first officer to use the flyPad.
```
Eliminate contraction

![First Officer EFB](assets/efb-firstofficer.png){loading=lazy}

> Our new PFD includes EIS S14 features with improve behaviours in numerous places. We have also made it lighter on frames improving performance in the simulator.

Reviewer: ```suggestion
Our new PFD includes EIS S14 features with improved behaviour in numerous aspects. We have also made it lighter on frames, improving performance in the simulator.
```
Fix typo, improve wording, add comma

> While there are just too many features to outline please check out our full changelog below!

Reviewer: ```suggestion
As there are too many features to outline here, please check out our full changelog below for more information.
```
Improve wording and tone
